ibm/service/vpc/resource_ibm_is_snapshot.go

```diff
@@ -18,30 +18,31 @@ import (
 )
 
 const (
-	isSnapshotName            = "name"
-	isSnapshotResourceGroup   = "resource_group"
-	isSnapshotSourceVolume    = "source_volume"
-	isSnapshotSourceImage     = "source_image"
-	isSnapshotUserTags        = "user_tags"
```

Comment:
Just to maintian the consistency with all vpc resources name the argument as "tags" (by default it indicates user tags across all resources in Terraform)

Reply:
made the corrections.